botocore/model.py

```diff
@@ -235,6 +236,13 @@ def operation_model(self, operation_name):
             raise OperationNotFoundError(operation_name)
         return OperationModel(model, self, operation_name)
 
+    def client_name_to_operation_name(self, name):
```

Comment:
Does this need to be in the models module?  Ideally the models should know nothing about clients, or even snake casing  Can we move this up to the client layer?